src/sentry_core.c

```diff
@@ -71,9 +71,20 @@ sentry_init(sentry_options_t *options)
         SENTRY_DEBUG("falling back to non-absolute database path");
         options->database_path = database_path;
     }
-    SENTRY_DEBUGF("Using database path \"%" SENTRY_PATH_PRI "\"",
+    SENTRY_DEBUGF("using database path \"%" SENTRY_PATH_PRI "\"",
         options->database_path->path);
 
+    sentry_path_t *handler_path = options->handler_path;
+    if (handler_path) {
```

Comment:
Would it make sense to move this block into the crashpad backend and add a comment why we're doing this? That's also where we check if the file exists, and so we can error out properly.